tests/golden/openshift4/crossplane/crossplane/01_helmchart/crossplane/crds/pkg.crossplane.io_providers.yaml

```diff
@@ -0,0 +1,331 @@
+null
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
No ArgoCD doesn't about `null` resources in the YAML stream. The `null` entries are generated because the Helm chart has an empty line above the first document separator for the CRDs, see e.g. https://github.com/crossplane/crossplane/blob/216d8c9b1ee10fe450cde7982eebffacc6724ee8/cluster/charts/crossplane/crds/apiextensions.crossplane.io_compositeresourcedefinitions.yaml#L1-L2